### PR TITLE
Pytave fixes

### PR DESCRIPTION
**Pull request by cbm755**
@genuinelucifer can you review this please?

### CODE REVIEW

File: inst/private/python_ipc_pytave.m [hunk 1]
```diff
@@ -70,7 +70,6 @@
 
   s = strjoin(cmd, newl);
   pyexec(s)
-  
-  info.raw = pyeval('str(_outs)');
```
Comment (genuinelucifer) [inline inst/private/python_ipc_pytave.m]: If we are removing info. We should remove `info.prelines` too (you probably missed it)

Comment (cbm755) [inline inst/private/python_ipc_pytave.m]: On 08/06/16 22:28, Abhinav Tripathi wrote:
> If we are removing info. We should remove |info.prelines| too (you
> probably missed it)

No I think its needed for error messages.  The error messages might be
wrong, but for now we need it.

Comment (genuinelucifer) [inline inst/private/python_ipc_pytave.m]: Oh ok. Actually I still don't know how this is used. I probably haven't
looked much into it.

Comment (cbm755) [inline inst/private/python_ipc_pytave.m]: > Actually I still don't know how this is used.

Its just a hack that @latot came up with: reports the line number an 
exception occurred within the `cmd` block of `python_cmd`.  But 
different IPC mechanisms added various amounts of stuff to `cmd`. 
`prelines` keeps track of how much stuff.  Its probably zero for PyTave ipc.

Rest of this look ok to merge?

Comment (genuinelucifer) [inline inst/private/python_ipc_pytave.m]: Hmm. Sure :)

**Pull request merged**